storage/scylla_table.h

```diff
@@ -81,6 +81,39 @@ status_t db_set_identity_id(db_identity_t* obj, cass_int64_t id);
  */
 cass_int64_t db_ret_identity_id(const db_identity_t* obj);
 
+/**
+ * @brief set time(seconds) in db_identity_t
+ *
+ * @param[in] obj pointer to db_identity_t
+ * @param[in] ts timestamp to be set into db_identity_t
+ * @return
+ * - SC_OK on success
+ * - SC_TA_NULL/SC_STORAGE_INVAILD_INPUT on error
+ */
+status_t db_set_identity_timestamp(db_identity_t* obj, cass_int64_t ts);
+
+/**
+ * @brief return time(seconds) in db_identity_t
```

Comment:
Not consistent to the form declared by `db_ret_identity_timestamp`. Should be `in seconds`.